src/server/db/sqlite3.test.ts

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2017 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import { Sqlite } from './sqlite3';
+import type { State, Server, LogEntry } from '../../types';
+
+describe('Sqlite', () => {
+  let db: Sqlite;
+
+  beforeAll(async () => {
+    db = new Sqlite();
+    await db.connect();
+  });
+  afterEach(async () => {
+    await db.clear();
+  });
+
+  test('basic', async () => {
+    // Must return undefined when no game exists.
+    const result = await db.fetch('matchID', { state: true });
+    expect(result.state).toEqual(undefined);
+
+    // Create game.
+    const state: unknown = { a: 1 };
+    const metadata: Server.MatchData = {
+      gameName: 'tic-tac-toe',
+      players: {},
+      createdAt: null,
+      updatedAt: null,
+    };
+
+    await db.createMatch('matchID', {
+      initialState: state as State,
+      metadata: metadata,
+    });
+
+    //Must return created game.
+    {
+      const result = await db.fetch('matchID', {
+        state: true,
+        metadata: true,
+        initialState: true,
+      });
+      expect(result.state).toEqual({ a: 1 });
+      expect(result.initialState).toEqual(result.state);
+      expect(result.metadata).toEqual({
+        gameName: 'tic-tac-toe',
+        setupData: undefined,
+        gameover: undefined,
+        players: {},
+        unlisted: null,
+        nextMatchID: null,
+        createdAt: null,
+        updatedAt: null,
+      });
+    }
+
+    // // Must return all keys
+    const keys = await db.listMatches();
+    expect(keys).toEqual(['matchID']);
```

Comment:
which is 'matchId', I see it repeated in several values. 
We should have a constant where the name of the constant gives meaning to what it is. Beyond its value